ReadingRainbow/MyBooks.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  ContentView.swift
+//  ReadingRainbow
+//
+//  Created by Tyler Sun on 7/20/23.
+//
+
+import SwiftUI
+import ComposableArchitecture
+
+struct MyBooksReducer: Reducer {
+    
+    struct State: Equatable {
+        @PresentationState var myBookDetail: BookDetailReducer.State?
+        var path = StackState<BookDetailReducer.State>()
+    }
+    
+    enum Action: Equatable {
+        case bookDetail(PresentationAction<BookDetailReducer.Action>)
+        case path(StackAction<BookDetailReducer.State, BookDetailReducer.Action>)
+    }
+    
+    var body: some ReducerOf<Self> {
+        Reduce { state, action in
+            switch action {
+                
+            case .bookDetail:
+                return .none
+            case .path:
+                return .none
+            }
+        }
+        .forEach(\.path, action: /Action.path) {
+            BookDetailReducer()
+        }
+        
+    }
+}
+
+
+
+struct MyBooks: View {
+    let store: StoreOf<MyBooksReducer>
+
+    let columns = [GridItem(.adaptive(minimum: 150))]
+ 
+    var body: some View {
+        NavigationStackStore(self.store.scope(state: \.path, action: { .path($0) })) {
+            WithViewStore(store, observe: { $0 }) { ViewStore in
+                ScrollView {
+                    LazyVGrid(columns: columns, spacing: 20) {
```

Comment:
This might be nice to pull into a separate View because we'll be reusing it later on.